README.md

```diff
@@ -1 +1,7 @@
-# basic-express-server
\ No newline at end of file
+# basic-express-server
+
```

Comment:
Should be using the [Lab template README](https://github.com/codefellows/seattle-code-javascript-401d41/blob/main/reference/LAB-README-TEMPLATE.md)